### ISSUE TITLE
Consider adding a "minikube open-dashboard" command.

### ISSUE THREAD

**Issue by dlorenc** [kubernetes/minikube#156]
The dashboard is running by default, but to access it the user has to figure out the NodePort or start a proxy. It would be easier if we just provided a command that opened the browser to the correct URL (or printed out the URL).

**Comment by luxas** [comment]
I guess we want something like this: https://github.com/toqueteos/webbrowser

**Comment by dlorenc** [comment]
A few different proposals:

minikube open-dashboard (opens a browser to the URL, using something like https://github.com/pkg/browser)
minikube open-dashboard --no-browser (just prints the URL instead of opening it)

Or the reverse:

minikube dashboard (just prints the URL)
minikube dashboard --open-in-browser (opens in the browser)

Or we could make it more generic:
minikube open-service <service name>

(but then we'd also have to deal with namespaces...)

**Comment by dlorenc** [comment]
I'm leaning toward the first option:

```shell
minikube open-dashboard
minikube open-dashboard --no-browser
```

**Comment by dlorenc** [comment]
cc @vishh any thoughts?

**Comment by vishh** [comment]
I prefer the default to be opening in a browser. How about dropping the `open` keyword and instead making it `minikube dashboard [--url]`?

**Comment by dlorenc** [comment]
Fixed with #170
